gilded-rose-kata/src/main/kotlin/interfaces/QualityFactory.kt

```diff
@@ -0,0 +1,11 @@
+package interfaces
+
+import enums.*
+
+interface QualityFactory {
+
+    var quality: Int
+
+    fun changeQualityValue(modifier: Modifier): Int
```

Comment:
Why a `QualityFactory` _changes_ de quantity value? I was expecting a `QualityFactory` to _create_ a quality..

Reply:
Makes absolute sense. Will change.